src/runtime/reference.ts

```diff
@@ -101,6 +113,7 @@ export abstract class ClientReference extends Reference {
       {
         id: Entity.id(entity),
         creationTimestamp: Entity.creationTimestamp(entity),
+        expirationTimestamp: Entity.creationTimestamp(entity),
```

Comment:
don't we need expirationTimestamp(entity)?

Reply:
good catch, updated, thanks.